_assignments/mp1.md

```diff
@@ -0,0 +1,395 @@
+
+Set the position of the child sphere to be $$(2,0,0)$$, which is four units from the parent sphere on the X-axis. For why this is, think about the scaling of the parent sphere.
+
+## Text
+
+> Put large text on a wall, listing your NetIDs and the details of the controls (which you will make later).
+Feel free to experiment with what you can put on a canvas, but keep it grandmother appropriate!
+
+Check out the Unity tutorial on [Creating Worldspace UIs](https://docs.unity3d.com/Manual/HOWTO-UIWorldSpace.html). Create a text canvas by going to `GameObject` → `UI` → `Text`. This will create a Unity Canvas, and some text on that canvas as a child of the canvas. It may also happen to be massive. Not what we want.
+
+To remedy this, select the canvas (not the text) from the hierarchy view. Its inspector should look like this:
+
+![Canvas Inspector](/img/assignments/mp1/image2.png)
+
+The first thing to do is change the Render mode from `Screen Space - Overlay`, to `World Space`. This changes our canvas from a UI element that is glued to the camera, to an object that is stationary in the world. Traditional UIs do not work well in VR, and we highly, highly advise against sticking any UI elements to the camera in your future MPs and Projects. Always attach UI elements to something in the world (See [this](https://developer.oculus.com/design/latest/concepts/bp-vision/)).
+
+Now that the canvas is a world space object, we can make it a more reasonable size. However, since the "Rect Transform"’s width and height determine the resolution of our text canvas, we cannot set them to be, say, $$5 \times 5$$, because then our text resolution would be 5 pixels by 5 pixels. Set the width and height to 1000 (that is $$1000 \times 1000$$ pixels). Shrink the canvas by setting the scale. We want our canvas to be 10 units by 10 units, and be $$1000 \times 1000$$ pixels, so our scaling is $$\frac{10}{1000} = 0.01$$. Make sure your text’s Rect Transform has the same width and height as its parent canvas, but leave the scale as 1. Place your canvas against one of the walls. You want to place your text ever so slightly (like, 0.001) in front of the wall it is against to avoid Z-fighting (which is where two objects have the same depth, and Unity can’t figure out which one to render). Below is an example of Z-fighting-
+
+![Z-fighting](/img/assignments/mp1/image24.png)
+
+Now, you can set your text color, size, font, width, whether it wraps or overflows, etc. Make your text has you and your partner’s NetIDs, as well as the controls for your game (more on this later). Make sure it is big enough for us to read. If the text appears blurry or jagged, then increase the width and height of the canvas and text (to increase the resolution), and scale them down further.
+
+## Store Asset
+
+> Import at least one free asset from the [Unity Store](https://www.assetstore.unity3d.com/) and place it in the room. You will need a free Unity account for this.

```

Comment:
you might need to move this up to provide instructions for importing `Oculus` from the store